### PR TITLE
docs. added universal javascript text.

### LINKED ISSUE #150: why electrode
Because we focus on the universal side, I think it might make sense to explain why we focus on that, for people to understand what the benefits are?

### PR DESCRIPTION
**Pull request by raydecastro**
- resolves #150.

### CODE REVIEW

File: _docs/01.2-why_use_electrode.markdown [hunk 1]
```diff
@@ -10,6 +10,19 @@ At [@WalmartLabs](http://www.walmartlabs.com/), we recently migrated our web sit
...
 might sound familiar if you've worked on a large, complex web app before.
 
+### Universal JavaScript
+Universal JavaScript allows us to use the same view rendering code from the 
+server and client. When a user requests a page, they immediately receive 
+the fully rendered HTML on initial page load. They will not have to wait 
+for the client side JavaScript code to load and render the page. 
+
+At the same time, when they change and navigate across different routes or 
+get new data, the client side code will handle all the rendering of the views.
```
Comment (caoyangs) [inline _docs/01.2-why_use_electrode.markdown]: This is not true when every page is server side rendered.

Comment (raydecastro) [inline _docs/01.2-why_use_electrode.markdown]: @caoyangs updated the text.

File: _docs/01.2-why_use_electrode.markdown [hunk 2]
```diff
@@ -10,6 +10,21 @@ At [@WalmartLabs](http://www.walmartlabs.com/), we recently migrated [walmart.co
 stack using React/Node.js. Along the way, we hit a few stumbling blocks that
 might sound familiar if you've worked on a large, complex web app before.
 
+### Universal JavaScript
+Universal JavaScript allows us to use the same view rendering code from the 
+server and client. When a user requests a page, they immediately receive 
+the fully rendered HTML on initial page load. They will not have to wait 
+for the client side JavaScript code to load and render the page. 
+
+Rich app like experiences are possible without requiring a full page load
```
Comment (caoyangs) [inline _docs/01.2-why_use_electrode.markdown]: what is `rich app`?

Comment (raydecastro) [inline _docs/01.2-why_use_electrode.markdown]: @caoyangs similar to https://en.wikipedia.org/wiki/Rich_Internet_application

**Pull request merged**